src/components/Recipes/Recipes.tsx

```diff
@@ -18,7 +18,7 @@ const Recipes: React.FC = () => {
 
   useEffect(() => {
     if(ingredientList.length !== 0) {
-      
+
```

Comment:
Just as a note - we probably need to revisit setting debounce for the onclick handler instead of fetch state — not a needed adjustment for now until we test in browser